src/Logging/index.js

```diff
@@ -0,0 +1,29 @@
+function logInfo(message) {
+  if (typeof newrelic !== 'undefined') {
+    newrelic.addPageAction('INFO', { message }); // eslint-disable-line no-undef
+  }
+}
+
+function logError(error) {
+  if (typeof newrelic !== 'undefined') {
+    newrelic.noticeError(error); // eslint-disable-line no-undef
+  }
+}
+
+function logAPIErrorResponse(error) {
+  let { message } = error;
+  if (error.response) {
```

Comment:
I'm curious where you're getting these variations of errors. Do you have a link to some documentation? It seems mostly inline with axios's error handling documentation: https://github.com/axios/axios#handling-errors.